book/src/super-sql/operators/tail.md

```diff
@@ -36,7 +36,8 @@ tail 2
 _Grab last two values of arbitrary sequence, using a different representation of two_
 ```mdtest-spq
 # spq
-tail 1+1
+const ONE = 1
+tail ONE+1
```

Comment:
@mccanne enhanced the example in the `head` doc to use this `const` approach so I figured we might as well do the same in `tail`.